.github/workflows/deploy-to-eks.yaml

```diff
@@ -58,13 +75,25 @@ jobs:
       - name: Setup juju
         run: |
           juju add-k8s kubeflow --client
-          juju bootstrap --no-gui kubeflow kubeflow-controller
+          juju bootstrap kubeflow kubeflow-controller
```

Comment:
Should we rename the cloud to `eks`? I know that in the past we did this but right now, it looks more clear that it's the cloud name and essentially unrelated to kubeflow. We can keep `kubeflow` as the model name.